src/seedu/addressbook/data/person/Address.java

```diff
@@ -6,27 +6,22 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address extends Contact{
 
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
 
-    public final String value;
-    private boolean isPrivate;
+    public final String value = "0";
```

Comment:
What's the point of assigning "0" to value? What's the need to maintain the variables value and isPrivate when you have it in the superclass?